README.md

```diff
@@ -184,12 +184,6 @@ This hook is installed automatically when using the script myEnvDFL.sh with any
 ## Dynaflow-launcher Documentation
 You can download Dynaflow-launcher documentation [here](https://github.com/dynawo/dynaflow-launcher/releases/download/v1.3.1/DynaflowLauncherDocumentation.pdf).
 
-## Get involved!
```

Comment:
In this case let's add a link toward the contributing file in the section added above

Reply:
normally is done in last commit